vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/SpaceLink.java

```diff
@@ -35,7 +37,8 @@
     @JoinColumn(name = "target_space_id")
     @NotFound(action=NotFoundAction.IGNORE)
     private ISpace targetSpace;
-
+    
+    
```

Comment:
remove changes in this file since they don't seem to be doing anything

Reply:
I am not sure about ```private ISpace targetSpace;``` whether it's being used or not, because I didn't add this code for the ticket. Should I remove it anyway?

Reply:
I only mean changes to the files that you made that don't do anything like new lines, etc.

Reply:
Please avoid changing parts of files that are no actual code changes (like this new empty lines).

Reply:
?

Reply:
?